CONTRIBUTING.md

```diff
@@ -63,13 +69,27 @@ Great, so you want to contribute. Let's get started:
 ## Getting Started with docker
 
 If you do not wish to install MySQL and Postgres locally to run unit tests
-can use [docker-compose](https://docs.docker.com/compose/) which will start
-both database, install all development dependencies and run all unit tests.
+install [docker-compose](https://docs.docker.com/compose/) for your platform.
+
+1. Download all images, build and start development container:
+
+    ```
+    docker-compose run --rm phinx
+    ```
 
-To get started, just run `docker-compose run --rm phinx`. It will download all
-images, build & start development container and switch you to it. So just install dependencies with `composer update --prefer-lowest` and run unittests with `vendor/bin/phpunit`.
+1. Install dependencies:
+
+    ```
+    composer update --prefer-lowest
```

Comment:
Why the prefer-lowest? That is weird. Usually this is only done in CI to ensure BC to those versions promised, but should not be part of normal dev and local default testing.

Reply:
That's true, I just keep the commands of the old version of this file, but it is a good opportunity to change it, I will remove it.